### PR TITLE
Drop language validation, enable language = "native"

### PR DESCRIPTION
**Pull request by dpprdan**
For the reasons mentioned in #90, this drops the validation of language tags and thus adds support for `language = "native"`. 

It also completes #89 in that it adds support for `NA`s for arguments other than `placename` and `latitude`/`longitude`.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -21,7 +22,7 @@ The new features include:
 
 ## Minor changes
 
-* The column name for both `languagecodes` and `countrycodes` is now `code`, and not `alpha2` and `Code`, respectively. 
+* The column name for `countrycodes` is now `code`, not `Code`.
```
Comment (maelle) [inline NEWS.md:25]: I'm wondering whether such changes should be "minor breaking changes" but this is a minor comment.

File: tests/testthat/test-oc_process.R [hunk 2]
```diff
@@ -323,3 +322,63 @@ test_that("oc_process handles various other arguments.", {
...
+    language = NULL,
+    countrycode = NULL,
+    min_confidence = NULL,
+    no_annotations = NULL,
+    no_dedupe = NULL,
+    no_record = NULL,
+    abbrv = NULL,
+    add_request = NULL
+  )
+  expect_match(res_null[[1]], "^((?!key=).)*$", perl = TRUE)
```
Comment (maelle) [inline tests/testthat/test-oc_process.R:343]: :open_mouth: I am impressed by the regex :smile_cat:

**Review by maelle — APPROVED** [review]

File: tests/testthat/test-oc_process.R (hunk 2 above)
Comment (dpprdan) [reply tests/testthat/test-oc_process.R:343]: Me too! 😲

File: NEWS.md (hunk 1 above)
Comment (dpprdan) [reply NEWS.md:25]: Would you consider the second point a "minor breaking change" as well (I wouldn't, I think)? Or would that be two sections then, "minor breaking changes" and "minor changes"?

Comment (maelle) [reply NEWS.md:25]: Yeah maybe?

Comment (dpprdan) [reply NEWS.md:25]: How about making this a non-minor change and adding a `name` column to make it easier to find the right `countrycode`? From e.g. https://datahub.io/core/country-list#data

**Pull request merged**